core/server/master/src/main/java/alluxio/master/file/meta/UfsSyncUtils.java

```diff
@@ -34,6 +34,8 @@ private UfsSyncUtils() {} // prevent instantiation
    */
   public static SyncPlan computeSyncPlan(Inode inode, Fingerprint ufsFingerprint,
       boolean containsMountPoint) {
+    // TODO(jiacheng): We still store a string in the inode instead of a Fingerprint obj.
+    //  This parse is unavoidable until we change it.
```

Comment:
Left a todo for my future self

Reply:
let's create an issue instead of a todo?

Reply:
https://github.com/Alluxio/alluxio/issues/15211, will update the comment